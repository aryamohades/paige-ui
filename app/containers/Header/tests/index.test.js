import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import { AppHeader } from '../index';

describe('<AppHeader />', () => {
  it('should match the snapshot if user is not authenticated', () => {
    const renderedComponent = renderer
      .create(
        <Router>
          <AppHeader />
        </Router>,
      )
      .toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });

  it('should match the snapshot if user is authenticated', () => {
    const renderedComponent = renderer
      .create(
        <Router>
          <AppHeader isAuthenticated />
        </Router>,
      )
      .toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
