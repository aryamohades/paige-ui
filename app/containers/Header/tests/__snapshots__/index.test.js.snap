// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppHeader /> should match the snapshot if user is authenticated 1`] = `
<div>
  <div
    className="NavBar-dbmxhD ldvbpF"
  >
    <div>
      <a
        className="HeaderBrand__StyledLink-Bfuoh ftwZgv"
        href="/"
        onClick={[Function]}
      >
        warehouse
      </a>
      <a
        className="itemStyles__StyledLink-fAIfBS gvqALX"
        href="/technologies"
        onClick={[Function]}
      >
        <span>
          Technologies
        </span>
      </a>
      <a
        className="itemStyles__StyledLink-fAIfBS gvqALX"
        href="/companies"
        onClick={[Function]}
      >
        <span>
          Companies
        </span>
      </a>
      <a
        className="itemStyles__StyledLink-fAIfBS gvqALX"
        href="/jobs"
        onClick={[Function]}
      >
        <span>
          Jobs
        </span>
      </a>
    </div>
    <div />
  </div>
</div>
`;

exports[`<AppHeader /> should match the snapshot if user is not authenticated 1`] = `
<div>
  <div
    className="NavBar-dbmxhD ldvbpF"
  >
    <div>
      <a
        className="HeaderBrand__StyledLink-Bfuoh ftwZgv"
        href="/"
        onClick={[Function]}
      >
        warehouse
      </a>
      <a
        className="itemStyles__StyledLink-fAIfBS gvqALX"
        href="/technologies"
        onClick={[Function]}
      >
        <span>
          Technologies
        </span>
      </a>
      <a
        className="itemStyles__StyledLink-fAIfBS gvqALX"
        href="/companies"
        onClick={[Function]}
      >
        <span>
          Companies
        </span>
      </a>
      <a
        className="itemStyles__StyledLink-fAIfBS gvqALX"
        href="/jobs"
        onClick={[Function]}
      >
        <span>
          Jobs
        </span>
      </a>
    </div>
    <div>
      <a
        className="itemStyles__StyledLink-fAIfBS gvqALX"
        href="/login"
        onClick={[Function]}
      >
        <span>
          Login
        </span>
      </a>
      <a
        className="itemStyles__ButtonLinkPrimary-iZgrZs hcCsOr"
        href="/register"
        onClick={[Function]}
      >
        <span>
          Register
        </span>
      </a>
    </div>
  </div>
</div>
`;
